app/models/drupal_users.rb

```diff
@@ -11,6 +11,7 @@ class DrupalUsers < ActiveRecord::Base
   has_many :answers, :foreign_key => :uid
   has_many :answer_selections, :foreign_key => :user_id
   has_many :drupal_comments, :foreign_key => 'uid'
+  has_one :location_tag, :foreign_key => 'uid', :dependent => :destroy
```

Comment:
Here, I thought we had decided that location tags would be associated with a profile tag, not with a person. But if there's a strong advantage to a `has_one` here, I'm open to it. 

If we do associate with a user record, we should use `user` and not `drupal_user`, which we're trying to deprecate.